fdbclient/Metacluster.cpp

```diff
@@ -77,6 +79,74 @@ json_spirit::mObject ClusterUsage::toJson() const {
 	return obj;
 }
 
+TenantMapEntry::TenantMapEntry(MetaclusterTenantMapEntry metaclusterEntry)
+  : tenantName(metaclusterEntry.tenantName), tenantLockState(metaclusterEntry.tenantLockState),
+    tenantGroup(metaclusterEntry.tenantGroup), configurationSequenceNum(metaclusterEntry.configurationSequenceNum) {
+	setId(metaclusterEntry.id);
```

Comment:
I think this line should could choose not to call `setId` if the ID is less than 0

Reply:
Instead, you would assign to `id` directly